dockers/docker-platform-monitor/start.sh.j2

```diff
@@ -37,6 +37,12 @@ if [ -e /usr/share/sonic/platform/fancontrol ]; then
     supervisorctl start fancontrol
 fi
 
+# If this platform has a python api wheel file, install it
+if [ -e /usr/share/sonic/platform/sonic_platform-1.0-py2-none-any.whl ]; then
```

Comment:
Hi @jleveque 
updated the start.sh script with the logic you provided. Verified the cases as well.

Reply:
@jleveque @sridhar-ravindran 
On other platforms, the sonic-platform whl may have different names, which fails this piece of code. It doesn't impact the normal function but generates a piece of log like:

    INFO pmon#supervisord: start.sh Error: Unable to locate /usr/share/sonic/platform/sonic_platform-1.0-py2-none-any.whl

To resolve this I have some suggestions:
1. Make this piece code vendor-specific.
2. Make `SONIC_PLATFORM_WHEEL` and `pip show sonic-platform` vendor-specific
3. remove the message "Error: Unable to locate ${SONIC_PLATFORM_WHEEL}"

Reply:
@stephenxs: I think it would be great if we could standardize this across all platforms. I can understand that different vendors may want to increment the version of their package, but I don't see a reason why anything else in the package name must be different. Maybe we could simply find the package by looking for a package matching `/usr/share/sonic/platform/sonic_platform-*-py2-none-any.whl`?